Services/UserService/Controllers/UserServiceController.cs

```diff
@@ -1,22 +1,20 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using DTO;
-using UserService.Interfaces;
 using UserService.Commands;
 
 namespace UserService.Controllers
 {
     [ApiController]
+    [Route("[controller]")]
     public class UserServiceController : Controller
     {
         
-        [Route("[controller]/post")]
+        [Route("createUser")]
         [HttpPost] 
         /// <summary>
         /// This method will be implemented in communication with other services
         /// </summary>
-        public string CreateUser([FromQuery] string email, string password)
+        public string CreateUser([FromServices] CreateUserCommand command, [FromBody] string email, [FromBody] string password)
```

Comment:
you can not have 2 body in one request, rework

Reply:
fixed